### DISCUSSION TITLE
nuclei -update not working on kali

### DISCUSSION THREAD

**Discussion by Red-Bug-7** [projectdiscovery/nuclei#1723] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/99800084/158574219-b26a6383-d867-4b20-88a7-c8df8b98b70f.png)
![image](https://user-images.githubusercontent.com/99800084/158574355-9238c955-bb99-4b53-899c-66372456316c.png)
Not working nuclei -update command on kali linux. When I give that command its shows the help page. And -update command is not listed in the UPDATE section. Please help.

**Comment by ehsandeep** [comment, marked as answer]
@Red-Bug-7 this is because the kali team has[ removed/patched](https://gitlab.com/kalilinux/packages/nuclei/-/blob/kali/master/debian/patches/Disable-update.patch) the `-update` support from nuclei, so if you using kali and installed using `apt`, you need to install or update nuclei using `go install` or download the binary from the [release](https://github.com/projectdiscovery/nuclei/releases) page.

**Reply by Red-Bug-7** [reply]
Installed with this command: `go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest`

Then how do I have to update? Please help

**Reply by forgedhallpass** [reply]
After installing nuclei using the `go install ...` command, the `-update` flag should work as expected, just make sure you are using the right binary. You can use the `which` command for that. The new nuclei binary should be under `$GOPATH/bin/nuclei`.

**Reply by Red-Bug-7** [reply]
![image](https://user-images.githubusercontent.com/99800084/158732698-6917e170-6b9c-406e-8155-79b2d97d1164.png)

![image](https://user-images.githubusercontent.com/99800084/158732666-6d909ccf-ac77-42a3-ac16-f459dff5f530.png)

`mv /usr/bin/nuclei ~/go/bin` After this command I gave `nuclei -update` and its updated to the new version. Thank you.
